qurro/_metadata_utils.py

```diff
@@ -73,6 +74,12 @@ def read_metadata_file(md_file_loc):
 
        This treats all metadata values (including the index column) as
        strings, due to the use of dtype=object.
+
+       NOTE THAT THIS WILL CONVERT ""s to np.NaN values in the output
```

Comment:
I'm assuming you mean blank spaces represented as `""` in the TSVs? Would clarify this.